<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Watcher</class>
 <widget class="QDialog" name="Watcher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>277</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Xpeccy WUTcha</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,10">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
     <item>
      <widget class="QLabel" name="label_15">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(107, 203, 255);</string>
       </property>
       <property name="text">
        <string>CPU</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="regGrid"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,10">
     <item>
      <widget class="QLabel" name="label_14">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(107, 203, 255);</string>
       </property>
       <property name="text">
        <string>MEMORY</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="10,0">
       <item>
        <widget class="QTableView" name="wchMemTab">
         <property name="minimumSize">
          <size>
           <width>255</width>
           <height>0</height>
          </size>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::ActionsContextMenu</enum>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="gridStyle">
          <enum>Qt::DotLine</enum>
         </property>
         <attribute name="horizontalHeaderMinimumSectionSize">
          <number>100</number>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>100</number>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderMinimumSectionSize">
          <number>17</number>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>17</number>
         </attribute>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLineEdit" name="wchBank0">
           <property name="minimumSize">
            <size>
             <width>75</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="frame">
            <bool>false</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="wchBank1">
           <property name="minimumSize">
            <size>
             <width>75</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="frame">
            <bool>false</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="wchBank2">
           <property name="minimumSize">
            <size>
             <width>75</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="frame">
            <bool>false</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="wchBank3">
           <property name="minimumSize">
            <size>
             <width>75</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="frame">
            <bool>false</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actAddWatcher">
   <property name="icon">
    <iconset resource="../xpeccy.qrc">
     <normaloff>:/images/add.png</normaloff>:/images/add.png</iconset>
   </property>
   <property name="text">
    <string>Add watcher</string>
   </property>
  </action>
  <action name="actDelWatcher">
   <property name="icon">
    <iconset resource="../xpeccy.qrc">
     <normaloff>:/images/cancel.png</normaloff>:/images/cancel.png</iconset>
   </property>
   <property name="text">
    <string>Delete watcher</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../xpeccy.qrc"/>
 </resources>
 <connections/>
</ui>
